WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -52,9 +52,20 @@
       <% end %>
     <% end %>
 
+    <% if !@competition.warnings.empty? %>
+      <% @competition.warnings.each do |key, value| %>
+        <div class="alert alert-warning">
+        <%= key %>
+        <%= value %>
+        </div>
+      <% end %>
+    <% end %>
+
     <% if @competition_admin_view || !is_actually_confirmed %>
       <%= f.input :id %>
+      <div class="form-group <%= !@competition.warnings[:name].nil? ? "has-warning" : "" %>">
```

Comment:
Oh wow, this works? I though devise was already generating these divs for us.

Can you include a screenshot in the PR?